### DISCUSSION TITLE
Suggestion: Detach fork from original repo

### DISCUSSION THREAD

**Discussion by jorsol** [codehaus-plexus/plexus-archiver#203] [Ideas]
I think this fork should be detached from the original repository ([krosenvold/plexus-archiver](https://github.com/krosenvold/plexus-archiver)) and turned into a standalone repo.

The steps should be straightforward, choose "Detach a fork" from the [GitHub Support request](https://support.github.com/request/fork), follow the Virtual Assistant, and that's all.

https://support.github.com/request/fork

**Comment by olamy** [comment]
I just created a ticket to detach all forks from the organisation https://github.com/orgs/codehaus-plexus/
